src/components/DrawerComponents.js

```diff
@@ -120,6 +121,12 @@ const DrawerComponents = ({ history, ...props }) => {
       to: "/profile-settings",
       icon: <PersonIcon />,
     },
+    {
+      key: "add-accommodation",
+      text: "Add accommodation",
+      to: "/add-accommodation",
+      icon: <AddToPhotosIcon />,
+    },
```

Comment:
a user should also be able to add rooms to the accommodation she or he created

Reply:
@blaise82 isn't that another feature on it's own? considering the implementation of both feature on the backend, what do you think?

Reply:
in the backend, it was one feature.
and I tried to check the PT we don't have that story for adding rooms

Please read your story again https://www.pivotaltracker.com/story/show/170947402

Reply:
@blaise82 On the backend we have two separate features, one for adding accommodations and another for adding rooms check the documentation. And the way they are implemented when creating the room we need it's accommodations(as a foreign key in the rooms table) so my question is if the feature is implemented as the story suggest do you think it will work that way? let me hear your thoughts on that.